index.html

```diff
@@ -1,41 +1,187 @@
-        </ul>
-    </div>
-</div> 
-</section>
+            <a href="#" class="fas fa-bars"></a>
+    </header>
+    <section>
+        <div class="home-content">
+            <h1>Hey There. <br>I’m Nonhlanhla Mndebele </h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product, feature or website look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <ul class="social">
+                <li class="github-a">
+                    <a href="#">
+                        <img src="./assets/icons/git.png" alt="img">
+                    </a>
+                </li>
+                <li class="linkedin-a">
+                    <a href="#">
+                        <img src="./assets/icons/i.png" alt="img">
+                        <span>
+                            <img src="./assets/icons/n.png" alt="img">
+                        </span>
+                    </a>
+                </li>
+                <li class="angellist a">
+                    <a href="#">
+                        <img src="./assets/icons/two.png" alt="img">
+                    </a>
+                </li>
+                <li class="twitter a">
+                    <a href="#">
+                        <img src="./assets/icons/twet.png" alt="img">
+                    </a>
+                </li>
+                <li class="medium a">
+                    <a href="#">
+                        <img src="./assets/icons/med.png" alt="img">
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="recent-works">
+        <h1>My Recent Works</h1>
+        <hr>
+        <div class="work-list">
+            <div class="work-item">
+                <img src="./assets/images/plc.png" alt="img">
+                <h1>Multi-Post Stories</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.</p>
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrp</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+            <div class="work-item-alt">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
```

Comment:
- [ ] Kindly add padding to your `work-item-alt` class so as to look exactly like the Figma design
   Your design :point_down: 
       
![Screenshot from 2022-05-31 11-55-29](https://user-images.githubusercontent.com/66097371/171158585-84aae356-27cd-46af-b2b0-d396770e6c5e.png)
  Figma design :point_down: 
    
![Screenshot from 2022-05-31 12-00-01](https://user-images.githubusercontent.com/66097371/171158936-e5d351b2-eb6d-4af1-ace4-8305a22de45b.png)
- [ ] also align the `see project` tag to the center